lib/govuk_index/document_type_inferrer.rb

```diff
@@ -1,5 +1,5 @@
 module GovukIndex
-  class DocumentTypeInferer
+  class DocumentTypeInferrer
```

Comment:
:man_facepalming:

Reply:
[At least it's easy to change](https://www.quora.com/What-is-the-origin-of-HTTP-header-%E2%80%9Creferer%E2%80%9D-misspelling) 😁